src/scripts/main.js

```diff
@@ -357,4 +357,27 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the HTML. If it doesn't, `document.querySelector('.dashboard')` will return `null`, causing an error when attempting to append children to it.